doc/examples.rst

```diff
@@ -19,8 +19,9 @@ generated list of the standard examples that come with SfePy.
 Example Applications
 --------------------
 
-(Note: the examples below were created around 2010 and do not reflect the
-current state of SfePy).
+- `Homogenization of piezoelectric porous media <http://sfepy.org/sfepy_examples/example_poropiezo/>`_ 
+- `Fish heart model <https://sfepy.org/fish_heart>`_
+- `Phononic materials <https://sfepy.org/phononic>`_
 
-- Fish heart model: https://sfepy.org/fish_heart
-- Phononic materials: https://sfepy.org/phononic
+(Note: *Fish heart model* and *Phononic materials* were created around 2010 and
+do not reflect the current state of SfePy).
```

Comment:
What about adding a year of "publication" to each example in the list above:

- `Homogenization of piezoelectric porous media <http://sfepy.org/sfepy_examples/example_poropiezo/>`_  (2020)
- `Fish heart model <https://sfepy.org/fish_heart>`_ (2010)
- `Phononic materials <https://sfepy.org/phononic>`_ (2010)

Reply:
Good idea!